src/components/Reactions/ReactionsList.tsx

```diff
@@ -32,6 +36,40 @@ export type ReactionsListProps<
   reverse?: boolean;
 };
 
+const ButtonWithTooltip = ({
+  children,
+  onMouseEnter,
```

Comment:
How will users be able to pass a custom callback, if it is not forwarded inside `UnMemoizedReactionsList`?

Reply:
Fair point but I'd rather have this button fully replaceable (passing down a custom component) instead of just passing down these two handle functions.